docs/samples/pmml/pmml.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "serving.kubeflow.org/v1alpha2"
+kind: "InferenceService"
+metadata:
+  name: "pmml-demo"
+spec:
+  default:
+    predictor:
+      pmml:
+        storageUri: "gs://anyisalin-model-store/pmml"
```

Comment:
uploaded the model to `gs://kfserving-examples/models/pmml` can update here correspondingly

Reply:
@yuzisun  Do you need help uploading?

Reply:
> @yuzisun Do you need help uploading?

I already uploaded to the above gcs bucket.

Reply:
Thanks, I have changed.